### PR TITLE
fix 9560 (recent added IDs), added 9462, 9461

### PR DESCRIPTION
**Pull request by lgariv**
issue #25 fixed.

### CODE REVIEW

File: itlwm/mac80211.cpp [hunk 1]
```diff
@@ -3201,6 +3201,16 @@ typedef void *iwm_match_t;
 #define    PCI_PRODUCT_INTEL_WL_9560_4    0x30DC        /* Dual Band Wireless AC 9560 */
 #define    PCI_PRODUCT_INTEL_WL_9560_5    0x271C        /* Dual Band Wireless AC 9560 */
 #define    PCI_PRODUCT_INTEL_WL_9560_6    0x271B        /* Dual Band Wireless AC 9560 */
+#define    PCI_PRODUCT_INTEL_WL_9462_1    0x42a4        /* Dual Band Wireless AC 9462 */
```
Comment (lgariv) [inline itlwm/mac80211.cpp:3204]: Added 9462 with PCI ID 8086:0x42a4

**Comment by lgariv** [comment]
> i dont have 9462 device to test and confirm, can any one can help to test these ids?

I can confirm for 8086:42a4

**Comment by zxystd** [comment]
@lgariv Hi, I found id `0x02a4` is not a Network Card PID as you added, can you tell me how did you get these ids? Fully thanks for your contribution.
image

**Comment by lgariv** [comment]
> @lgariv Hi, I found id `0x02a4` is not a Network Card PID as you added, can you tell me how did you get these ids? Fully thanks for your contribution.
> 
> image
> 
> 

I don't remember, to be honest; but I think it was from an open source Linux driver. Can't remember the name

**Pull request merged**